### ISSUE TITLE
Review of 4.3.1 WCAG Conformance Requirements

### ISSUE THREAD

**Issue by rickj** [w3c/epub-specs#914]
In the second paragraph, the word "as" just prior to the link to [WCAG 2.0] should be "and"

In the second paragraph, the last sentence is confusing with two references to "specification".  For clarity, where the sentence starts with "In fact, any EPUB specification, and even..." should be changed to "In fact, any specific EPUB requirement, and even..."

**Comment by mattgarrish** [comment]
The "as" looks fine to me. That they require Level AA is more than the requirement in the specification, so that's the influence they typically exert. 

I'd suggest we drop the talk about other specifications, as it's neither here nor there as far as legal ramifications go. Getting into requirements seems more confusing unless we talk about how specifications can set should/must and Level A/AA for conformance. The specification is designed to allow flexibility in conformance already, so we don't need to go there.

The scope being "vendors" is somewhat limiting, though, as it excludes procurement demands, which could similarly demand a higher level. Perhaps it should be:

"In fact, any procurer or vendor of EPUB Publications can set a higher conformance requirement than the baseline defined here."

**Comment by clapierre** [comment]
I agree with adding the sentence on higher conformance.

**Comment by mattgarrish** [comment]
If it helps, I've made it clearer that higher conformance is legislated as an example:

> Although this specification only requires Level A conformance, local and national laws can influence the level of conformance an EPUB Publication has to meet to be considered accessible. Level AA conformance is often cited as the benchmark for accessibility in legal frameworks and policies, for example. Additionally, any procurer or distributor of EPUB Publications can demand higher conformance requirements than the baseline defined here.
